mempool/collector.go

```diff
@@ -127,9 +127,12 @@ func (t *MempoolDataCollector) mempoolTxns() ([]exptypes.MempoolTx, txhelpers.Me
 		//fee, _ := txhelpers.TxFeeRate(msgTx)
 		// log.Tracef("tx fee: GRM result = %f, msgTx = %f", tx.Fee, fee.ToCoin())
 
+		_, feeRate := txhelpers.TxFeeRate(msgTx)
```

Comment:
Regarding the comment a few lines up, we may need special  handling for coinbase transactions.  I haven't checked.

Reply:
Can there be a case where a coinbase TX gets added to the mempool?

Reply:
Nah, but I think stakebase/votes might have the same issue as they're generating coins too.

Reply:
FeeRate is displayed for SSTX and regular TXs but not for votes; weird `feeRate` values coming from coinbase/stakebase txs won't be shown.  If you think it's warranted I can check `feeRate` coming back from `txhelpers.TxFeeRate` and maybe blank it out - what do you think?

Reply:
Yeah, true.  It's good for this.